README.md

```diff
@@ -5,22 +5,295 @@
+## Getting Started
+
+* Import Realm.
+    ```dart
+    import 'package:realm/realm.dart';
+    ```
+
+* Define a data model class `_Car`.
+    ```dart
+    @RealmModel()
+    class _Car {
+      late String make;
+
+      late String model;
+    
+      int? kilometers = 500;
+    }
+    ```
+
+* Generate Generates RealmObject class `Car` from data model class `_Car`.
+
+    ```
+    flutter run realm generate
+    ```
+
+* Open a Realm and add some objects.
+
+    ```dart
+    var config = Configuration([Car.schema]);
+    var realm = Realm(config);
+
+    var car = Car("Telsa", "Model Y", kilometers: 5);
+    realm.write(() {
+      realm.add(car);
+    }
+    ```
+
+* Query objects in Realm.
+
+    ```dart
+    var cars = realm.all<Car>();
+    Car myCar = cars[0];
+    print("My car is ${myCar.make} model ${myCar.model}");
+
+    cars = realm.all<Car>().query("make == 'Tesla'");
+    ```
+
+## Samples
+
+For complete samples check the [Realm Flutter and Dart Samples](https://github.com/realm/realm-dart-samples).
+
+## Documentation
+
+For API documentation go to 
+ * [Realm Flutter API Docs](https://pub.dev/documentation/realm/latest/)
+
+ * [Realm Dart API Docs](https://pub.dev/documentation/realm_dart/latest/)
+
+For a complete documentation go to [Realm Flutter and Dart SDK Docs](https://docs.mongodb.com/realm/sdk/flutter/).
+
+
+## Limitations
+
+* This version of Realm Flutter and Dart SDK allows working with a local only (on device) Realm database in Flutter and Dart desktop. Realm Sync functionality is not implemented.
+
+* It provides the functionality for creating, retrieving, querying, sorting, filtering, updating Realm objects and supports change notifications.
+
+* Flutter Desktop on Linux is not yet support
+
+# Realm Flutter SDK 
+
+The Realm Flutter package name is `realm`
+
+## Environment setup for Realm Flutter
+
+* Supported platforms are Flutter (iOS, Android, Windows, MacOS) and Dart standalone (Windows, MacOS and Linux)
+
+* Flutter ^2.8.0
+
+## Usage
+
+* Add `realm` package to a Flutter application.
+    ```
+    flutter pub add realm
+    ```
+
+* Enable generation of RealmObjects.
+
+    * Add `build_runner` package to `dev_dependencies`.
+        ```
+        dart pub add build_runner --dev
+        ```
+    * Enable `realm_generator` by adding a `build.yaml` file to the application.
+        ```yaml
+        targets:
+            $default:
+                builders:
+                    realm_generator|realm_object_builder:
+                        enabled: true
+                        generate_for:
+                            - lib/*.dart 
+        ```
+
+    * Import Realm in a dart file (ex. `catalog.dart`).
+        ```dart
+        import 'package:realm/realm.dart';
+        ```
+
+    * Declare a part file `catalog.g.dart` in the begining of the `catalog.dart` dart file after all imports.
```

Comment:
Maybe here we need a very short explanation that part file 'catalog.g.dart' will contain generated files. We have it below, but on this step is not clear why we are declaring part file.

Reply:
Lets leave it as it is. this is made clear on the generation step. Less words to read is better.